enricher/ecs/ecs.go

```diff
@@ -55,13 +59,15 @@ func NewEnricher() *Enricher {
 // EnrichRecord modifies existing record.
 func (enr *Enricher) EnrichRecord(r map[interface{}]interface{}, t time.Time) map[interface{}]interface{} {
 	resource := map[interface{}]interface{}{
-		mappings.RESOURCE_ACCOUNT_ID: enr.canvaAWSAccount,
-		"service.name":               enr.canvaAppName,
```

Comment:
Thanks for fixing the inconsistencies here.

Nit: it would be more readable to have them in separate PRs. Note there are more inconsistencies here: some variables are hard-coded here, others are coming from mappings. We could maybe have a separate PR to make them more consistent.